CHANGES.rst

```diff
@@ -1,3 +1,48 @@
+0.7.0 (??-10-2020)
+------------------
+
+This releases is a new major release with some new features and some backwards
+incompatible changes that require a careful upgrade and some manual
+intervention. The 0.7.x releases will also be that last to support Python 2. If
+you haven't upgraded to Python 3 yet, we advise you to do now.
+
+**BC break** The major change in this version is no longer initiating the
+`skosprovider.registry.Registry` on starting the application, but when a
+request is created. The previous way of working created problems with
+SQLAlchemy providers in a webserver using mutiple threads. Please review the
+docs at
+https://atramhasis.readthedocs.io/en/latest/customisation.html#creating-conceptschemes
+to see how it works now. For more background, have a look at the
+pyramid_skosprovider_ docs at https://pyramid-skosprovider.readthedocs.io/en/0.9.0/install.html
+
+- All requirements were updated to their latest versions. Python versions were
+  fixed to 2.7, 3.6, 3.7 and 3.8. If you made custom changes, you might have to
+  edit them. (#508, #519, #513, #566)
+- Instantiation of the SKOS registry was changed to work on a per request
+  basis. (#346, #490, #535)
+- Fixed a major issue with generating the expanded version of a concept. By
+  default the assumption was that concepts in a collection were also narrower
+  concepts of the collection's superordinate concept, but the implementation
+  for this was incomplete and contained buggy. This has been changed to an
```

Comment:
> was incomplete and contained buggy.

best replace with "was incomplete and buggy" or "was incomplete and contained bugs"